src/platform/plugins/shared/dashboard/public/dashboard_api/load_dashboard_api/load_dashboard_api.ts

```diff
@@ -25,7 +25,8 @@ export async function loadDashboardApi({
   getCreationOptions?: () => Promise<DashboardCreationOptions>;
   savedObjectId?: string;
 }) {
-  const creationStartTime = performance.now();
+  const creationStartTime = performance.getEntriesByName('dashboard_app_mount', 'mark')[0]
+    .startTime;
```

Comment:
This only works for the first dashboard loaded in a session. What happens if the user loads another dashboard? Then that dashboard load time would be pegged to application mount. Should `dashboard_app_mount` be cleared after the first load and all subsequent loads use `performance.now()`?